read me updated for model importing component (#564)
Azure/azureml-assets: common/README.md

# General Purpose Assets
## Overview

> This directory contains the general purpose assets that includes [components](https://docs.microsoft.com/en-us/azure/machine-learning/concept-component) and [environments](https://docs.microsoft.com/en-us/azure/machine-learning/concept-environments), which can be used for purpose like [model registration](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-models?view=azureml-api-2&tabs=cli%2Cuse-local) and [model deployment](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-deploy-online-endpoints?view=azureml-api-2&tabs=azure-cli) steps in [azure machine learning pipelines](https://learn.microsoft.com/en-us/azure/machine-learning/concept-ml-pipelines?view=azureml-api-2).

Reviewer: ```suggestion
This directory contains the general purpose assets that include [components](https://docs.microsoft.com/en-us/azure/machine-learning/concept-component) and [environments](https://docs.microsoft.com/en-us/azure/machine-learning/concept-environments) that can be used for purposes like [model registration](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-models?view=azureml-api-2&tabs=cli%2Cuse-local) and [model deployment](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-deploy-online-endpoints?view=azureml-api-2&tabs=azure-cli) steps in [Azure Machine Learning pipelines](https://learn.microsoft.com/en-us/azure/machine-learning/concept-ml-pipelines?view=azureml-api-2).
```

Author: Done

## Structure of this folder

> | Directory         | Description                                                                          |
> |:------------------|:-------------------------------------------------------------------------------------|
> | `components/deploy_model` | Definition of the component for the model deployment.                                                      |
> | `components/register_model`       | Definition for the component model registration.                                   |
> | `environments`  | Definition for the environment python-sdk-v2  |

Reviewer: Removed tabs and extra spaces to make maintenance easier.
```suggestion
| Directory | Description |
|:---|:---|
| `components/deploy_model` | Definition of the model deployment component. | 
| `components/register_model` | Definition for the model registration component. |
| `environments` | Definition for the python-sdk-v2 environment. |
```

Author: Done.

## Examples

> [Create pipeline using component assets registered in azureml](https://github.com/Azure/azureml-examples/blob/hrishikesh/workflow/sdk/python/foundation-models/system/import/import_model_into_registry.ipynb)

Reviewer: ```suggestion
[Create pipeline using component assets registered in Azure ML](https://github.com/Azure/azureml-examples/blob/hrishikesh/workflow/sdk/python/foundation-models/system/import/import_model_into_registry.ipynb)
```

Author: Done

> [Create pipeline using component assets registered in azureml](https://github.com/Azure/azureml-examples/blob/hrishikesh/workflow/sdk/python/foundation-models/system/import/import_model_into_registry.ipynb)

Reviewer: @ManojBableshwar, should examples like these be published elsewhere?
